getting-started/exploring-enterprise.md

```diff
@@ -0,0 +1,55 @@
+# Exploring TimescaleDB Enterprise
+
+TimescaleDB currently offers three product lines: OSS, Community, and Enterprise. 
+The Enterprise offering features automated data management that enables hands-free data retention policies 
+and data re-ordering on disk. For an overview of how TimescaleDB product lines differ, 
+please take a look at our [Pricing][pricing] page.
+
+## Requesting a License Key
+
+It is easy to get started with TimescaleDB Enterprise. We offer a free 30-day license 
+that you can request [online][license]. The license key will be sent to you 
+over email, and will unlock all Enterprise capabilities. 
+
+To upgrade or obtain a full Enterprise license key, please [contact sales][contact-sales].
+
+## Activating TimescaleDB Enterprise
+
+Before you can activate TimescaleDB Enterprise, you must have a Timescale-Licensed version of 
+TimescaleDB up and running. Note that our standard binary releases are by default Timescale-Licensed. 
+
+>:WARNING: If you explicitly built or downloaded an Apache 2.0 binary, you will not be able to 
+directly activate TimescaleDB Enterprise. You must upgrade to a Timescale-Licensed binary first. 
+Please note that this also applies for any TimescaleDB versions prior to v1.2.0.
+
+First connect to the PostgreSQL instance:
+
+```bash
+# Connect to PostgreSQL, using a superuser named 'postgres'
+psql -U postgres -h localhost
+```
+
+Next create a new empty database (skip this if you already have a database):
+
```

Comment:
The instructions say to create empty DB, but the command is just about setting license key.

Reply:
Thanks for catching that. Fixing.